File: python3/1523/Solution.py
class Solution:
    def countOdds(self, low: int, high: int) -> int:
        numCount = high - low + 1
        oddCount = numCount // 2

        if (
            numCount % 2 != 0 and
            low % 2 != 0
        ):
            oddCount += 1

        return oddCount

File: java/1523/Solution.java
class Solution {
    public int countOdds(int low, int high) {
        int numCount = high - low + 1;
        int oddCount = numCount / 2;

        if (numCount % 2 != 0 &&
            low % 2 != 0) {
            oddCount += 1;
        }

        return oddCount;
    }
}
